kernel/rros/clock.rs

```diff
@@ -1028,7 +1028,7 @@ impl FileOperations for ClockOps {
     }
 }
 
-pub fn clock_factory_dispose(ele: factory::RrosElement) {}
+pub fn clock_factory_dispose(ele: &mut factory::RrosElement) {}
```

Comment:
Great, separate the jobs making them clearer.

Reply:
Sorry, I forgot to mention one thing. The reason why the function parameters are modified here is because `fac.despose.unwrap()(self)` is used in line 298 of the new factory.rs, and then `self.remove_element_dev()` needs to be called, so the ownership cannot be taken away.